src/components/BulkEnrollmentModal/index.jsx

```diff
@@ -3,11 +3,16 @@ import PropTypes from 'prop-types';
 import {
   Field, reduxForm, SubmissionError, reset,
 } from 'redux-form';
+
+import { logError } from '@edx/frontend-platform/logging';
 import {
   Alert, Button, Icon, Modal,
+
```

Comment:
nittiest of nits, don't bother unless you're making more changes; extra line here.